src/Tizen.NUI.Scene3D/src/public/Controls/Camera.cs

```diff
@@ -0,0 +1,731 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Scene3D
+{
+    /// <summary>
+    /// Enumeration for type determination of how the camera operates.
+    /// </summary>
+    /// This will be released at Tizen.NET API Level 10, so currently this would be used as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum CameraType
+    {
+        /// <summary>
+        /// Camera orientation is taken from the CameraActor.
+        /// </summary>
+        FreeLook,
+        /// <summary>
+        /// Camera is oriented to always look at a target.
+        /// </summary>
+        LookAtTarget
```

Comment:
FreeLook, LookAtTarget both use "look" in other positions which are ugly in my opinion..
how about named enum to CameraFocusType or CameraPointType sor more better to call it just FocusType as ProjectionMode has no prefix "Camera".
and elements just named Free and OnTarget?
it is simply my personal preference so if this names are already used in DALi, you can ignore my comments.

Reply:
I removed the option because other engines like unity, unreal, and ios works differently with the LookAt keyword.
So I will add similar feature in the next work.